chess/4-database/database.md

```diff
@@ -108,25 +108,80 @@ As part of your unit test deliverable you need to meet the following requirement
 1. Ensure that all of your unit tests work, including the new DAO tests and the Service tests you wrote in the previous assignment.
 
 > [!IMPORTANT]
-> You must place your data access test cases in a folder named server/src/test/java/dataaccess.
+> You must place your data access test cases in a folder named `server/src/test/java/dataaccess`.
 
 ### Code Quality
 
-For this phase the TAs will grade the quality of your project's source code. The rubric used to evaluate code quality can be found here: [Rubric](../code-quality-rubric.md)
+For this phase the auto grader will grade the quality of all your project's source code.
+The rubric used to evaluate code quality can be found here: [Rubric](../code-quality-rubric.md).
+You can also test your quality with the auto grader independent of a specific phase submission.
+
+![Code Quality](../codeQuality.png)
+
+### Common Submission Issues
+
+Since you're adding code that acts with an external program in Phase 4, the likelihood of having auto-grader test
+output different from your local output is higher than previous phases.
+
+If you encounter this, here are some common things to try (expand the collapsible section).
+
+<details>
+
+<summary>Tips for resolving AutoGrader submission issues when local tests pass</summary>
+
+> [!IMPORTANT]
```

Comment:
This alert and the following do not render properly within the collapsed section. See this discussion for the reported issue: [GitHub Community discussion #143379](https://github.com/orgs/community/discussions/143379).

Any recommendations on the next best option?

Reply:
This issue has been avoided by extracting the collapsible section into a separate page.